android-tests/build.gradle

```diff
@@ -0,0 +1,61 @@
+buildscript {
+    repositories {
+        jcenter()
+    }
+
+    dependencies {
+        // Android Gradle plugin
+        classpath 'com.android.tools.build:gradle:1.3.1'
+
+        // Resolves android-sdk dependencies
+        classpath 'com.jakewharton.sdkmanager:gradle-plugin:0.12.0'
```

Comment:
It's for developers, I guess @akarnokd used it without noticing :)

On Fri, Nov 13, 2015, 07:47 Jake Wharton <notifications@github.com> wrote:

> In android-tests/build.gradle
> <https://github.com/ReactiveX/RxJava/pull/3519#discussion_r44749555>:
>
> > @@ -0,0 +1,61 @@
> > +buildscript {
> > +    repositories {
> > +        jcenter()
> > +    }
> > +
> > +    dependencies {
> > +        // Android Gradle plugin
> > +        classpath 'com.android.tools.build:gradle:1.3.1'
> > +
> > +        // Resolves android-sdk dependencies
> > +        classpath 'com.jakewharton.sdkmanager:gradle-plugin:0.12.0'
>
> Not needed. .travis.yml takes care of this.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ReactiveX/RxJava/pull/3519/files#r44749555>.
>
-- 

@artem_zin